templates/memeooorr.yaml

```
@@ -1,6 +1,6 @@
 name: "Memeooorr"
 description: "Memeooorr @twitter_handle"
-hash: bafybeichcqlatpqn4ifklszf2odsllna76nm4du6irva3favj5f2wzsaju
+hash: bafybeickejdjgbq3ofib4k26qsff6vhc7sfktq73yyg7dyj2var4qk5bsa
 image: https://gateway.autonolas.tech/ipfs/QmQYDGMg8m91QQkTWSSmANs5tZwKrmvUCawXZfXVVWQPcu
 service_version: v0.4.0-alpha7
```

can we update the version as well please here and in serviceTemplate
```suggestion
service_version: v0.4.1-alpha1
```